extension-localization/src/main/java/com/mapbox/maps/extension/localization/Localization.kt

```diff
@@ -52,6 +75,37 @@ internal fun setMapLanguage(locale: Locale, style: StyleInterface, layerIds: Lis
     }
 }
 
+fun checkServerSideLocalizationSet(listener: (Boolean) -> Unit) {
+  // create worker thread to run settingsInterface methods.
+  val workerThread = HandlerThread(
+    "SETTINGS_THREAD", THREAD_PRIORITY_DEFAULT
```

Comment:
```suggestion
    "LOCALIZATION_THREAD", THREAD_PRIORITY_DEFAULT
```
also nice to move it to constants